### PR TITLE
add hdai back

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Description
A few sentences describing the overall goals of the pull request's commits.

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: definitions/v3/hdai.yml [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        3: 2 # 2x
+        4: 2 # 2x free
+        6: 2 # 2x 50% free
+        "*": 1
+    minimumratio:
+      text: 1.0
+    minimumseedtime:
+      # 2 day (as seconds = 2 x 24 x 60 x 60)
+      text: 172800
+# json NexusPHP v1.5 beta 5 20120707
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:223]:
```suggestion
# json NexusPHP v1.5 beta 5 20120707

```

File: definitions/v3/hdai.yml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+  rows:
+    selector: data.items
+    count:
+      selector: data.total
+
+  fields:
+    category:
+      selector: category
+    source:
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:111]:
```suggestion
    _source:
```

File: definitions/v3/hdai.yml [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+      selector: source
+      case:
+        0: ""
+        1: CN
+        2: EU/US
+        3: HK/TW
+        4: JP
+        5: KR
+        6: Other
+    medium:
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:121]:
```suggestion
    _medium:
```

File: definitions/v3/hdai.yml [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+        1: "UHD Blu-ray"
+        2: "Blu-ray"
+        3: "Remux"
+        4: "WEB-DL"
+        5: "Encode"
+        6: "HDTV"
+        7: "DVD"
+        8: "CD"
+        9: "MiniBD"
+    codec:
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:134]:
```suggestion
    _codec:
```

File: definitions/v3/hdai.yml [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+      case:
+        0: ""
+        1: "H.265(HEVC)"
+        2: "H.264"
+        3: "VC-1"
+        4: "Xvid"
+        5: "MPEG-2"
+        7: "MPEG-4"
+        6: "Other"
+    audiocodec:
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:145]:
```suggestion
    _audiocodec:
```

File: definitions/v3/hdai.yml [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+        6: "LPCM"
+        7: "FLAC"
+        8: "WAV"
+        9: "APE"
+        10: "AAC"
+        11: "AC3"
+        12: "OGG"
+        13: "MPEG"
+        14: "Other"
+    standard:
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:163]:
```suggestion
    _standard:
```

File: definitions/v3/hdai.yml [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+      selector: name
+    details:
+      selector: details
+    download:
+      selector: download
+    description:
+      selector: small_descr
+      filters:
+        - name: append
+          args: " [{{ .Result.medium }} {{ .Result.standard }} {{ .Result.codec }} {{ .Result.audiocodec }} {{ .Result.source }}]"
```
Comment (ilike2burnthing) [inline definitions/v3/hdai.yml:183]:
```suggestion
          args: " [{{ .Result._medium }} {{ .Result._standard }} {{ .Result._codec }} {{ .Result._audiocodec }} {{ .Result._source }}]"
```

**Review by ilike2burnthing — CHANGES_REQUESTED** [review]
Should be the only changes needed

**Comment by bakerboy448** [comment]
TY - nice catches

weird the validation didn't catch those

**Comment by bakerboy448** [comment]
@Qstick why is the validation not failing as expected / the validation check is existing successfully when it finds an issue?

**Pull request closed without merging**